static/css/custom.css

```diff
@@ -1,3 +1,15 @@
+/* logo center */
+.icon img {
```

Comment:
I think this is the wrong solution. 

I believe the error came when i introduced the “img” block in custom.css, which was meant to fix images embedded in the cards. It’s supposed to be “p img” not “img” … and i believe this y-offset can be removed.

Reply:
Then again it still looks like its not centered to me (i just pushed to main) so maybe this offset is necessary